Docs/src/tutorials/basictutorials/code/tutorialapplicationcpp.md

```diff
@@ -0,0 +1,131 @@
+# Tutorial application startup source file {#tutorialapplicationcpp}
```

Comment:
this just duplicates the Bootstrap sample, so we should drop it

Reply:
Yes it is. But for sake of study (and Basic Tutorial 1 suggest it) person moves along the declaration of entities and learns things. 
We could not drop it since the person has to start with something.

Reply:
why not start with bootstrap?

Reply:
In my opinion it iw better not know about bootstrap while one learning Ogre for first time. It is nice to have it for reference and for someone who knows a bit about engine.
Reading bootstrap code one can be just overwhelmed by its complexity. It is better to introduce concepts step by step. Original tutorials have this notion. And for me as first time learner of it it made positive.

Another point is we leaving room for keeping idea with tutorials as with basic application in case if bootstrap suddenly changed to be not appropriate for tutorials. But it is just subjective.